examples/e07_sample_bot_structure/Cargo.toml

```
@@ -5,14 +5,14 @@ authors = ["my name <my@email.address>"]
 edition = "2018"
 
 [dependencies]
-dotenv = "0.15"
-tracing = "0.1"
-tracing-subscriber = "0.2"
+dotenv = "0.15" 
+tracing = "0.1" 
+tracing-subscriber = "0.2" 
```

Trailing whitespace is unnecessary.
```suggestion
dotenv = "0.15"
tracing = "0.1"
tracing-subscriber = "0.2"
```